mita/async.py

```diff
@@ -21,6 +21,9 @@ def bootstrap_pecan(signal, sender):
         config_path = os.path.abspath(os.path.join(here, '../config/config.py'))
 
     pecan.configuration.set_config(config_path, overwrite=True)
+    # Once configuration is set we need to initialize the models so that we can connect
+    # to the DB wth a configured mapper.
+    models.init_model()
```

Comment:
Do you need ``models.start()`` here as well? Looks like you had it in tasks.py before.